src/app/pages/rpc-stx-transfer-sip10-ft/rpc-stx-transfer-sip10-ft.utils.ts

```diff
@@ -66,6 +67,10 @@ function getTransactionOptionsFromRpcRequest(address: string) {
       : undefined,
     nonce: decodedRpcRequest.params.nonce,
     sponsored: decodedRpcRequest.params.sponsored,
+    postConditionMode:
+      decodedRpcRequest.params.postConditionMode === 'allow'
+        ? PostConditionMode.Allow
+        : PostConditionMode.Deny,
```

Comment:
Do we ever want this to be controlled by the app? For a SIP10 transfer, we can safely always set allow mode right?

Reply:
Yeah, I think you are right ...I believe I had it that way before refactoring, thx.

Reply:
I think you mean `deny` mode or we show this... before I just didn't set it so defaults to `deny` and shows the post conditions.

![Screenshot 2025-05-13 at 11 15 56 AM](https://github.com/user-attachments/assets/8fafa128-917f-401d-bcc6-0535a7fee90e)

Reply:
In the case of the sub-transaction types we allow, e.g. sip10, the intention is to transfer a SIP10 token. 

What outcome might an app want to associate with this transfer, other than the transfer of the described SIP10?